draco/asp/helpers.lp

```diff
@@ -4,3 +4,21 @@
 attribute(mark_channel,M,C) :-
     entity(encoding,M,E),
     attribute((encoding,channel),E,C).
+
+% @helper(mark_scale_channel) Whether a mark has a scale for a channel.
```

Comment:
I think this helper is not quite correct. We could have a scale defined on the view and the mark. This rule implies that we would have two scales for the encoding. We should only use the closest scale definition to the encoding.